fluster/decoders/chromium.py

```diff
@@ -0,0 +1,70 @@
+# Fluster - testing framework for decoders conformance
+# Copyright (C) 2021, Fluendo, S.A.
+#  Author: Manuel Jimeno Martínez <mjimeno@fluendo.com>, Fluendo, S.A.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Library General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Library General Public License for more details.
+#
+# You should have received a copy of the GNU Library General Public
+# License along with this library; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+# Boston, MA 02111-1307, USA.
+from abc import ABC
+from pathlib import Path
+
+from fluster.codec import Codec, OutputFormat
+from fluster.decoder import Decoder, register_decoder
+from fluster.http.http_custom_server import HTTPServerHandler
+
+
+class ChromiumDecoder(Decoder, ABC):
+    """Generic class for Chromium decoder"""
+    description = "This is an implementation for the Chromium decoder"
+
+
+@register_decoder
+class ChromiumH264(ChromiumDecoder):
+    """H264 class for Chromium decoder"""
+    provider = 'Chromium'
```

Comment:
Chromium is open source. Provider is Chromium.

Provider field is not needed for a `Decoder`.  Therefore, we can delete it if doubts.

Reply:
What if we don't register the decoder if fluster_chromium is not available? Instead of the current behavior, that is first register and then test if available.

Reply:
What is the problem with leaving it like this? If the class we registered for does not exist, what do we do?

- Shall we give an error message and stop the execution? Fluster is no longer functional, 99% of the time will not install
- Do we not say anything? Then, when we try to use it, it will say that it is not registered, and we will not know why it does not work. So we have not given the error.

Reply:
I meant, that if fluster-chromium is not available, then when you list the Fluster's decoders, the ChromiumH264 decoder will not be listed. You can print a warning if you want. 

I prefer that behavior, but it is not very important.

Reply:
Done.